src/pool.c

```diff
@@ -541,21 +544,21 @@ hsk_pool_resolve(
     return HSK_SUCCESS;
   }
 
-  hsk_name_req_t *head = hsk_map_get(&peer->names, req->hash);
```

Comment:
`peer->names` is list of `tlds` instead, right ?

Reply:
yeah the names we have requested proofs for from a given peer.